<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctor</class>
 <widget class="QMainWindow" name="doctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>319</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
border-image: url(:/search/deletesquarebutton_79500.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>378</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>621</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>160</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="surnameLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>surname</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Пошук</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>467</x>
      <y>320</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
 border-image: url(:/search/1.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>393</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
     min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string>Редагувати</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>566</x>
      <y>0</y>
      <width>21</width>
      <height>451</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Chancery L</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Лікар</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>30</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
     min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string>Головне меню</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_patient">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>561</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
 QTableView QTableCornerButton::section {
     background: red;
     border: 2px outset red;
 }
QTableView {
    border: 2px solid #3873d9;
    border-top-color: #4287ff;
    border-radius: 4px;
    background-color: #fff;
    background-image: url(:/img/other/background.png);
    gridline-color: #777;
    selection-background-color: #ccdfff;
    color:#333;
    font-size:12px;
	font: 75 oblique 11pt &quot;Tlwg Typist&quot;;
 }
</string>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Імя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Прізвище</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Палата</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>393</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
     min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string>Інформація</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>320</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     border-image: url(:/search/search.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>315</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     border-image: url(:/search/REFRESH2_26887.png);

	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>30</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
     min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string>Виписані пацієнти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>393</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
     min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string>Картка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>392</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
     min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string>Добавити запис</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>58</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
    min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string>Прийом</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>310</y>
      <width>20</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>373</x>
      <y>324</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>358</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>362</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Палата:</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>346</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>358</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
 border-image: url(:/search/1.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>-31</x>
      <y>346</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
